packages/router/src/RouterLink.ts

```diff
@@ -83,6 +84,7 @@ export interface UseLinkDevtoolsContext {
   route: RouteLocationNormalized & { href: string }
   isActive: boolean
   isExactActive: boolean
+  error: string
```

Comment:
Small nitpick: `string | null` seems more appropriate

Reply:
I assume you intended for `null` to be used instead of `''` when there's no error. I've updated accordingly.